src/ddmd/backend/backconfig.c

```diff
@@ -51,7 +51,7 @@ void out_config_init(
                         // 2: fake it with C symbolic debug info
         bool alwaysframe,       // always create standard function frame
         bool stackstomp,        // add stack stomping code
-        bool avx,               // use AVX instruction set
+        unsigned char avx,      // use AVX instruction set (0, 1, 2)
```

Comment:
It's currently not an enum in the backend, but would be a nice follow-up.
https://github.com/dlang/dmd/blob/a336b69ca42ac535ea365fc4e70f84a7378d7e67/src/ddmd/backend/cdef.h#L908